src/classes/Keyboard.h

```diff
@@ -206,11 +209,12 @@ class LedKeyboard {
 		DeviceInfo currentDevice;
 		
 		#if defined(hidapi)
-			hid_device *m_hidHandle;
+			hid_device *m_hidHandle = NULL;
 		#elif defined(libusb)
 			bool m_isKernellDetached = false;
-			libusb_device_handle *m_hidHandle;
+			libusb_device_handle *m_hidHandle = NULL;
```

Comment:
This actually fixes a segfault in ListKeyboards when no compatibls devices are found (we call libusb_close with uninitialized variable)